eng/jobs/validate-signing.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  pool: {}
+  dependsOn: {}
+jobs:
+- job: Signing_Validation
+  # Run only if all build legs succeeded
+  condition: and(succeeded('Linux_x64_glibc'),
+                succeeded('Linux_Arm64'),
+                succeeded('Linux_Arm'),
+                succeeded('Linux_x64_Alpine36'),
+                succeeded('Linux_x64_Rhel6'),
+                succeeded('Windows_x64'),
+                succeeded('Windows_x86'),
+                succeeded('Windows_Arm'),
+                succeeded('Windows_Arm64'),
+                notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest'))
```

Comment:
Can you use this conditional to handle omitting the signing validation template?

https://github.com/dotnet/arcade/blob/master/azure-pipelines.yml#L28

```
${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Reply:
It is already taken care of here : https://github.com/dotnet/core-setup/pull/4860/files#diff-fec826feae04e51c0d94076385408bdcR428 . Agreed, it will be more readable if done here too.